fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -782,8 +783,17 @@ private void initAggregateBuiltins() {
                 if (t1.isFunctionType() || t1.isNull() || t1.isChar() || t1.isPseudoType()) {
                     continue;
                 }
-                addBuiltin(AggregateFunction.createBuiltin(MAX_BY,
-                        Lists.newArrayList(t1, t), t1, Type.VARBINARY, true, true, false));
+                addBuiltin(AggregateFunction.createBuiltin(
+                        MAX_BY, Lists.newArrayList(t1, t), t1, Type.VARBINARY, true, true, false));
+            }
+
+            // MIN_BY
+            for (Type t1 : Type.getSupportedTypes()) {
+                if (t1.isFunctionType() || t1.isNull() || t1.isChar() || t1.isPseudoType()) {
+                    continue;
+                }
```

Comment:
can it support nested type? like array, map, struct?

Reply:
Looks like it does not support, and there is no <> operator for array, map, struct.

Reply:
add `|| t1.isComplexType()`?